src/app/view/_routes/edit-organization/edit-organization.component.html

```diff
@@ -46,16 +46,46 @@
         <button mat-button matStepperNext type="button">Next</button>
       </div>
     </mat-step>
-    <mat-step formGroupName="2" [stepControl]="formArray?.get([2])" [optional]="true">
-      <ng-template matStepLabel>Admin management</ng-template>
-      <mat-form-field>
-        <mat-label>Manager</mat-label>
-        <input matInput formControlName="managerEmailCtrl" placeholder="Ex. mariorossi@gmail.com" />
+    <mat-step formGroupName="2" [optional]="true">
+      <ng-template matStepLabel>Administrator management</ng-template>
+      <mat-list>
+        <div *ngFor="let admin of administrators">
+          <mat-list-item class="fieldName">{{ admin.role }}</mat-list-item>
+          <mat-list-item class="fieldData">{{ admin.email }}</mat-list-item>
+          <button
+            mat-button
+            matTooltip="Remove from administrators"
+            aria-label="Remove this user from administrators"
+            type="button"
+            (click)="deleteAdmin(admin)"
+          >
+            <mat-icon>delete</mat-icon>
+          </button>
+          <mat-divider></mat-divider>
+        </div>
+      </mat-list>
+      <mat-form-field class="adminType">
+        <mat-label>Administrator type</mat-label>
+        <mat-select formControlName="adminRole">
+          <mat-option *ngFor="let admin of adminType" [value]="admin">
+            {{ admin }}
+          </mat-option>
+        </mat-select>
       </mat-form-field>
-      <mat-form-field>
-        <mat-label>Viewer</mat-label>
-        <input matInput formControlName="viewerEmailCtrl" placeholder="Ex. giorgiobianchi@gmail.com" />
+      <mat-form-field class="adminEmail">
+        <mat-label>New administrator</mat-label>
+        <input formControlName="adminEmail" matInput placeholder="Ex. mariorossi@gmail.com" />
```

Comment:
"Ex." è una abbreviazione che tendiamo a usare noi italiani, gli inglesi di solito se vogliono fare un esempio scrivono "e.g." che sta per exempli gratia. ma in realtà essendo già una convenzione qua e negli altri posti così puoi lasciare direttamente solo mariorossi@gmail.com, o address@example.com